src/main/resources/io/jenkins/plugins/opentelemetry/backend/ElasticBackend/config.jelly

```diff
@@ -1,15 +1,25 @@
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
-         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-         <f:entry title="Kibana base URL" field="kibanaBaseUrl" description="e.g. 'https://kibana.example.com:5601'">
-             <f:textbox />
-         </f:entry>
-         <f:advanced>
-             <f:entry title="Name" field="name">
-                 <f:textbox default="${descriptor.displayName}"/>
-             </f:entry>
-             <f:entry title="Metrics visualization URL Template" field="metricsVisualizationUrlTemplate">
-                 <f:textbox />
-             </f:entry>
-         </f:advanced>
+<j:jelly xmlns:j="jelly:core"
+         xmlns:f="/lib/form">
+    <f:entry title="Kibana base URL" field="kibanaBaseUrl" description="e.g. 'https://kibana.example.com:5601'">
+        <f:textbox/>
+    </f:entry>
+    <f:entry title="Kibana dashboard title" field="kibanaDashboardTitle"
+             description="Name of the Kibana dashboard used to monitor Jenkins. Default value is 'CI/CD Overview'">
```

Comment:
I guess it's now `Jenkins overview`?

Reply:
Correct. I'll rework this assistance to setup the Kibana dashboard.